src/components/form/formHead/formHead.js

```diff
@@ -1,18 +1,30 @@
 import React, { Fragment } from "react";
 import DropDownField from "../../dropDownField/dropDownField";
+import Shoe from "../../styled/shoe/shoe";
+import icon__scissor from "../../../../public/assets/icon__scissor.svg";
+import styled from "styled-components";
 
-// TODO add icons for shoe and Scissor
-const Shoe = () => <h3>shoe</h3>;
-const Scissor = () => <h3>scissor</h3>;
+const Width = styled.div.attrs({
+}) `
+	width: 10rem;
```

Comment:
Would it be worth having `<Scissor />` with a width prop like  `<Scissor width={width} />`, and then have the styled component `Width` written as 
```js
const Width = styled.div.attrs({
+}) `
+	width: {props => props.width || 10rem};
`
```
So that we don't have the `<Scissor />` svg component hard-coded at 10rem